src/programs/azimuthal_average/azimuthal_average.cpp

```diff
@@ -0,0 +1,1534 @@
+#include "../../core/core_headers.h"
+
+class
+	AzimuthalAverage : public MyApp
+{
+
+public:
+	bool DoCalculation();
+	void DoInteractiveUserInput();
+
+private:
+};
+
+void azimuthal_alignment(Image *input_stack, Image *input_stack_times_ctf, int number_of_images, int max_iterations, float unitless_bfactor, bool mask_central_cross, int width_of_vertical_line, int width_of_horizontal_line, float inner_radius_for_peak_search, float outer_radius_for_peak_search, float max_shift_convergence_threshold, float start_angle_for_peak_search, float end_angle_for_peak_search, float rotation_step_size, float max_rotation_convergence_threshold, float pixel_size, int number_of_frames_for_running_average, int savitzy_golay_window_size, int max_threads, float *x_shifts, float *y_shifts, float *psi_angles, float *ctf_sum_of_squares);
+void apply_ctf(Image *current_image, CTF ctf_to_apply, float *ctf_sum_of_squares, bool absolute);
+void divide_by_ctf_sum_of_squares(Image *current_image, float *ctf_sum_f_squares); // const float
```

Comment:
In principle, the text in the signature doesn't matter, but should float *ctf_sum_f_squares -> float *ctf_sum_of_squares

Reply:
Typo on my part, I will fix this. Also, I was thinking of making the array ctf_sum_of_squares a constant to prevent accidentally changing it in divide_by_ctf_sum_of_squares as good practice, as my comment reminds me to do so.